CRM/Export/BAO/Export.php

```diff
@@ -706,14 +706,14 @@ public static function exportComponents(
 
     $queryString .= $groupBy;
 
-    // always add contact_a.id to the ORDER clause
-    // so the order is deterministic
-    //CRM-15301
-    if (strpos('contact_a.id', $order) === FALSE) {
-      $order .= ", contact_a.id";
-    }
-
     if ($order) {
+      // always add contact_a.id to the ORDER clause
+      // so the order is deterministic
+      //CRM-15301
+      if (strpos('contact_a.id', $order) === FALSE) {
+        $order .= ", contact_a.id";
+      }
+      
```

Comment:
@danbrellis you need to remove the whitespace at this newline as stated my test build failure https://test.civicrm.org/job/CiviCRM-Core-PR/11180/checkstyleResult/new/